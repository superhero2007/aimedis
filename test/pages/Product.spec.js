import React from 'react';
import { shallow } from 'enzyme';

import Product from 'pages/Product';

function setup() {
  return shallow(<Product />);
}

describe('Product', () => {
  const wrapper = setup();

  it('should be a Component', () => {
    expect(wrapper.instance() instanceof React.Component).toBe(true);
  });

  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
