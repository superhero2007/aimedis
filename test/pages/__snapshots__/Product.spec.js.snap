// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product should render properly 1`] = `"<div class=\\"app__product app__route\\"><div class=\\"container\\"><section class=\\"app__product__section1\\"><div class=\\"app__product__carousel\\"><div class=\\"app__product__carousel__content\\"><div class=\\"app__product__carousel__content__inner\\"><div class=\\"app__product__carousel__content__inner__item\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__product__item\\"><div class=\\"app__product__item__content\\"><div class=\\"row\\"><div class=\\"col-12 col-lg-6 p-lg-0 app__product__item__content__info\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div><p class=\\"app__product__item__content__info__title\\">Your health data in one place.</p><div class=\\"app__product__item__content__info__image d-block d-lg-none\\"><img src=\\"\\" alt=\\"No Asset\\"/></div><p class=\\"app__product__item__content__info__description\\">Aimedis has been developed by doctors for patients, doctors and hospitals - designed to give patients the complete control over their health data and support.</p><div class=\\"app__product__item__content__info__button\\"><div class=\\"app__product__item__content__info__button__icon\\"><div class=\\"app__product__item__content__info__button__icon__triangle\\"></div></div><p class=\\"app__product__item__content__info__button__text\\">PLAY VIDEO</p></div></div></div></div><div class=\\"col-6 d-none d-lg-flex\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><img src=\\"\\" alt=\\"No Asset\\"/></div></div></div></div></div></div></div><div class=\\"app__product__carousel__content__inner__item prev\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__product__item\\"><div class=\\"app__product__item__content\\"><div class=\\"row\\"><div class=\\"col-6 d-none d-lg-flex direction-rtl\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><img src=\\"\\" alt=\\"No Asset\\"/></div></div><div class=\\"col-12 col-lg-6 p-lg-0 app__product__item__content__info\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div><p class=\\"app__product__item__content__info__title\\">Video chat with doctor.</p><div class=\\"app__product__item__content__info__image d-block d-lg-none\\"><img src=\\"\\" alt=\\"No Asset\\"/></div><p class=\\"app__product__item__content__info__description\\">Aimedis has been developed by doctors for patients, doctors and hospitals - designed to give patients the complete control over their health data and support.</p><div class=\\"app__product__item__content__info__button\\"><div class=\\"app__product__item__content__info__button__icon\\"><div class=\\"app__product__item__content__info__button__icon__triangle\\"></div></div><p class=\\"app__product__item__content__info__button__text\\">PLAY VIDEO</p></div></div></div></div></div></div></div></div></div><div class=\\"app__product__carousel__content__inner__item active\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__product__item\\"><div class=\\"app__product__item__content\\"><div class=\\"row\\"><div class=\\"col-12 col-lg-6 p-lg-0 app__product__item__content__info\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div><p class=\\"app__product__item__content__info__title\\">Independent Second Opinion.</p><div class=\\"app__product__item__content__info__image d-block d-lg-none\\"><img src=\\"\\" alt=\\"No Asset\\"/></div><p class=\\"app__product__item__content__info__description\\">Aimedis has been developed by doctors for patients, doctors and hospitals - designed to give patients the complete control over their health data and support.</p><div class=\\"app__product__item__content__info__button\\"><div class=\\"app__product__item__content__info__button__icon\\"><div class=\\"app__product__item__content__info__button__icon__triangle\\"></div></div><p class=\\"app__product__item__content__info__button__text\\">PLAY VIDEO</p></div></div></div></div><div class=\\"col-6 d-none d-lg-flex\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><img src=\\"\\" alt=\\"No Asset\\"/></div></div></div></div></div></div></div><div class=\\"app__product__carousel__content__inner__item next\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__product__item\\"><div class=\\"app__product__item__content\\"><div class=\\"row\\"><div class=\\"col-6 d-none d-lg-flex direction-rtl\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><img src=\\"\\" alt=\\"No Asset\\"/></div></div><div class=\\"col-12 col-lg-6 p-lg-0 app__product__item__content__info\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div><p class=\\"app__product__item__content__info__title\\">Video chat with doctor.</p><div class=\\"app__product__item__content__info__image d-block d-lg-none\\"><img src=\\"\\" alt=\\"No Asset\\"/></div><p class=\\"app__product__item__content__info__description\\">Aimedis has been developed by doctors for patients, doctors and hospitals - designed to give patients the complete control over their health data and support.</p><div class=\\"app__product__item__content__info__button\\"><div class=\\"app__product__item__content__info__button__icon\\"><div class=\\"app__product__item__content__info__button__icon__triangle\\"></div></div><p class=\\"app__product__item__content__info__button__text\\">PLAY VIDEO</p></div></div></div></div></div></div></div></div></div><div class=\\"app__product__carousel__content__inner__item\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__product__item\\"><div class=\\"app__product__item__content\\"><div class=\\"row\\"><div class=\\"col-12 col-lg-6 p-lg-0 app__product__item__content__info\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div><p class=\\"app__product__item__content__info__title\\">Independent Second Opinion.</p><div class=\\"app__product__item__content__info__image d-block d-lg-none\\"><img src=\\"\\" alt=\\"No Asset\\"/></div><p class=\\"app__product__item__content__info__description\\">Aimedis has been developed by doctors for patients, doctors and hospitals - designed to give patients the complete control over their health data and support.</p><div class=\\"app__product__item__content__info__button\\"><div class=\\"app__product__item__content__info__button__icon\\"><div class=\\"app__product__item__content__info__button__icon__triangle\\"></div></div><p class=\\"app__product__item__content__info__button__text\\">PLAY VIDEO</p></div></div></div></div><div class=\\"col-6 d-none d-lg-flex\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><img src=\\"\\" alt=\\"No Asset\\"/></div></div></div></div></div></div></div><div class=\\"app__product__carousel__content__inner__item\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__product__item\\"><div class=\\"app__product__item__content\\"><div class=\\"row\\"><div class=\\"col-6 d-none d-lg-flex direction-rtl\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><img src=\\"\\" alt=\\"No Asset\\"/></div></div><div class=\\"col-12 col-lg-6 p-lg-0 app__product__item__content__info\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div><p class=\\"app__product__item__content__info__title\\">Video chat with doctor.</p><div class=\\"app__product__item__content__info__image d-block d-lg-none\\"><img src=\\"\\" alt=\\"No Asset\\"/></div><p class=\\"app__product__item__content__info__description\\">Aimedis has been developed by doctors for patients, doctors and hospitals - designed to give patients the complete control over their health data and support.</p><div class=\\"app__product__item__content__info__button\\"><div class=\\"app__product__item__content__info__button__icon\\"><div class=\\"app__product__item__content__info__button__icon__triangle\\"></div></div><p class=\\"app__product__item__content__info__button__text\\">PLAY VIDEO</p></div></div></div></div></div></div></div></div></div></div><div class=\\"app__product__carousel__content__external d-block d-lg-none\\"><div class=\\"app__product__carousel__content__external__content\\"><div class=\\"app__product__carousel__content__external__content__item item1\\"></div><div class=\\"app__product__carousel__content__external__content__item item2\\"></div><div class=\\"app__product__carousel__content__external__content__item active item3\\"></div><div class=\\"app__product__carousel__content__external__content__item item4\\"></div><div class=\\"app__product__carousel__content__external__content__item item5\\"></div><div class=\\"app__product__carousel__content__external__content__item item6\\"></div></div></div></div></div></section><section class=\\"app__product__section2\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__product__section2__title text-center d-block d-lg-none\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__title\\"><p class=\\"app__title__subtitle\\">Platform Overview</p><p class=\\"app__title__title\\">Cool features</p></div></div></div><div class=\\"app__product__section2__content\\"><div class=\\"row\\"><div class=\\"col-12 col-lg-6\\"><p class=\\"app__product__section2__content__title d-none d-lg-block\\">Aimedis is mobile and with cool features</p><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__overview active\\"><div class=\\"app__overview__internal\\"><div class=\\"app__overview__internal__content\\"><div class=\\"app__overview__internal__content__order\\"><p class=\\"app__overview__internal__content__order__text body\\">1</p></div><div class=\\"app__overview__internal__content__paragraph\\"><p class=\\"app__overview__internal__content__paragraph__title title\\">Video chat with doctor.</p><p class=\\"app__overview__internal__content__paragraph__subtitle body\\">In every actual browser</p></div><div class=\\"app__overview__internal__content__next ml-auto\\"><span class=\\"i-long-arrow-right d-none d-lg-inline\\"></span><span class=\\"i-long-arrow-down d-inline d-lg-none\\"></span></div></div></div><div class=\\"app__overview__external\\"><div class=\\"app__overview__external__image\\"><img src=\\"\\" alt=\\"No Asset\\"/></div></div></div></div><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__overview\\"><div class=\\"app__overview__internal\\"><div class=\\"app__overview__internal__content\\"><div class=\\"app__overview__internal__content__order\\"><p class=\\"app__overview__internal__content__order__text body\\">2</p></div><div class=\\"app__overview__internal__content__paragraph\\"><p class=\\"app__overview__internal__content__paragraph__title title\\">Multilanguage</p><p class=\\"app__overview__internal__content__paragraph__subtitle body\\">also fully functional abroad</p></div><div class=\\"app__overview__internal__content__next ml-auto\\"><span class=\\"i-long-arrow-right d-none d-lg-inline\\"></span><span class=\\"i-long-arrow-down d-inline d-lg-none\\"></span></div></div></div><div class=\\"app__overview__external\\"><div class=\\"app__overview__external__image\\"><img src=\\"\\" alt=\\"No Asset\\"/></div></div></div></div><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__overview\\"><div class=\\"app__overview__internal\\"><div class=\\"app__overview__internal__content\\"><div class=\\"app__overview__internal__content__order\\"><p class=\\"app__overview__internal__content__order__text body\\">3</p></div><div class=\\"app__overview__internal__content__paragraph\\"><p class=\\"app__overview__internal__content__paragraph__title title\\">Easy to use</p><p class=\\"app__overview__internal__content__paragraph__subtitle body\\">also for unexperienced patients simple &amp; fast</p></div><div class=\\"app__overview__internal__content__next ml-auto\\"><span class=\\"i-long-arrow-right d-none d-lg-inline\\"></span><span class=\\"i-long-arrow-down d-inline d-lg-none\\"></span></div></div></div><div class=\\"app__overview__external\\"><div class=\\"app__overview__external__image\\"><img src=\\"\\" alt=\\"No Asset\\"/></div></div></div></div><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__product__section2__content__mobile d-none d-lg-flex\\"><div class=\\"app__product__section2__content__mobile__button\\"><img src=\\"\\" alt=\\"No Asset\\"/><div class=\\"app__product__section2__content__mobile__button__title\\"><p class=\\"app__product__section2__content__mobile__button__title__small\\">Available on the</p><p class=\\"app__product__section2__content__mobile__button__title__big\\">App Store</p></div></div><div class=\\"app__product__section2__content__mobile__button\\"><img src=\\"\\" alt=\\"No Asset\\"/><div class=\\"app__product__section2__content__mobile__button__title\\"><p class=\\"app__product__section2__content__mobile__button__title__small\\">Available on the</p><p class=\\"app__product__section2__content__mobile__button__title__big\\">Google Play</p></div></div></div></div></div><div class=\\"col-lg-6 d-none d-lg-block app__product__section2__content__mobile__image\\"><img src=\\"\\" alt=\\"No Asset\\"/></div></div></div></div></section><section class=\\"app__product__section3\\"><div class=\\"app__feature\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__feature__title\\">Discover the best features for you.</div><div class=\\"app__feature__content\\"><div class=\\"app__get__early__access\\"><input type=\\"text\\" class=\\"app__get__early__access__input\\" placeholder=\\"Your Email Address\\"/><button type=\\"button\\" class=\\"app__get__early__access__button btn btn__primary\\">Get <span class=\\"d-none d-lg-inline\\">Early </span>Access</button></div></div></div></div></section><section class=\\"app__product__mask d-none d-lg-block\\"><div class=\\"app__product__mask__two\\"><img src=\\"\\" alt=\\"No Asset\\"/></div><div class=\\"app__product__mask__four\\"><img src=\\"\\" alt=\\"No Asset\\"/></div><div class=\\"app__product__mask__five\\"><img src=\\"\\" alt=\\"No Asset\\"/></div></section></div></div>"`;
